Vagrantfile

```diff
@@ -36,7 +18,45 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         end
 
         test_1.vm.provision "ansible" do |ansible|
-            ansible.playbook = "vagrant/test_1.yml"
+            ansible.playbook = "vagrant/test_rhel.yml"
         end
     end
-end
\ No newline at end of file
+
+    # Define a test box to be scanned during tests
+    config.vm.define "test_2" do |test_2|
+          test_2.vm.network "private_network", ip: "192.168.50.11"
+          test_2.vm.host_name = "test2.example.com"
+
+          test_2.ssh.forward_x11 = true
+
+          test_2.vm.provider :libvirt do |domain|
```

Comment:
I'll remove this. I didn't really educate myself on Vagrant. Just essentially kept, what was there for the test endpoints. But I've tested with out this portion and it works. I'll also remove the libvrt section in the doc.